README.md

```diff
@@ -434,6 +434,15 @@ run_params:
       subnets: array of strings          // These should be in the same VPC and Availability Zone as your instance
       security_groups: array of strings  // These should be in the same VPC as your instance
       assign_public_ip: string           // supported values: ENABLED or DISABLED
+  task_placement:
+    strategy:
+    - type: string                       // Valid value: "random" -- field key not needed when strategy type is random
```

Comment:
This is a bit confusing for me to read/understand. So the only valid value is "random" and if the strategy is "random" then field is optional? That's how I read that.

Comparing this to the ECS Params code, I'm confused why type is there twice. How about this (if its even correct):
```
strategy:
  - type: string // Valid values: "spread"|"binpack"|"random"
     field: string // not needed if type is "random"
```

Reply:
Yeah, it was difficult to come up with how to explain this. I'm updating this also to say "not valid" vs "not needed", but I like your suggestion.